docs/user-guide/install.md

```diff
@@ -91,3 +91,60 @@ vsphere
 ID                  NAME                DESCRIPTION         ENABLED
 ```
 
+## Installation on Windows Docker Hosts
+
+vDVS plugin can be installed on Windows Server 2016/Windows 10 VMs using the PowerShell installer. You will need Docker EE/Docker for Windows version **17.03 or above** (with Windows containers mode enabled) on the VM.
+
+First, start an instance of PowerShell with the "Run as administrator" option.
+
+* **To download the plugin installer**
+
+```
+PS C:\> Invoke-WebRequest -Uri https://raw.githubusercontent.com/vmware/docker-volume-vsphere/master/install-vdvs.ps1 -Out install-vdvs.ps1
+```
+
+* **To install the plugin**
+
+```
+PS C:\> .\install-vdvs.ps1 https://bintray.com/vmware/vDVS/download_file?file_path=docker-volume-vsphere_0.16_windows_developer-preview.zip
```

Comment:
it is better to add a sample command as it keeps changing per release ... e.g. `PS C:\> .\install-vdvs.ps1 <windows_plugin bundle>`